csvkit/utilities/csvjson.py

```diff
@@ -92,11 +92,45 @@ def dump_json(data, newline=False):
             )
 
             features = []
+
+            global min_lon, min_lat, max_lon, max_lat
+
             min_lon = None
             min_lat = None
             max_lon = None
             max_lat = None
 
+            def update_boundary_lat(lat):
+                global min_lat, max_lat
+
+                if min_lat is None or lat < min_lat:
+                    min_lat = lat
+                if max_lat is None or lat > max_lat:
+                    max_lat = lat
+
+            def update_boundary_lon(lon):
+                global min_lon, max_lon
+
+                if min_lon is None or lon < min_lon:
+                    min_lon = lon
+                if max_lon is None or lon > max_lon:
+                    max_lon = lon
+
+            def update_boundary_coords(coords):
+                if (isinstance(coords, list) and len(coords) == 2 and
+                        isinstance(coords[0], float) and
+                        isinstance(coords[1], float)):
+                    update_boundary_lon(coords[0])
+                    update_boundary_lat(coords[1])
+                else:
+                    try:
```

Comment:
I didnt find any.  Removing.
(I did a quick check of types that I dont use using https://github.com/node-geojson/geojson-flatten/tree/master/test/fixture , resulting in https://github.com/wireservice/csvkit/issues/871 ).  If there are still some types which might cause an exception, we need exceptions here to catch and fix them.